### PR TITLE
usb: Make the cdc rx item more generic

### PR DESCRIPTION
**Pull request by arturo182**
For reuse in HID

### CODE REVIEW

**Comment by kbeckmann** [comment]
Oh a new PR!

File: app/drivers/usb.h [hunk 1]
```diff
@@ -1,9 +1,15 @@
 #pragma once
 
+#include "drivers/usb/def.h"
 #include "errors.h"
 
 #define EUSB_NO_INIT			(EUSB_BASE + 0)
 #define EUSB_USBD_INIT			(EUSB_BASE + 1)
 #define EUSB_USBD_START			(EUSB_BASE + 2)
 
+struct usb_rx_queue_item {
```
Comment (kbeckmann) [inline app/drivers/usb.h]: Maybe we should have an even more generic name. like usb_fs_data or something.

Comment (kbeckmann) [reply app/drivers/usb.h]: usb_fs_packet perhaps

**Comment by arturo182** [comment]
Let's rename later if we feel like it :P

**Pull request merged**